otter/auth.py

```diff
@@ -235,6 +235,12 @@ def __init__(self, identity_admin_user, identity_admin_password, url, admin_url)
 
     @wait(ignore_kwargs=['log'])
     def _auth_me(self, log=None):
+        def _log_failed_auth(err):
+            """
+            Log this as a string we know we can find in the logging feed
+            """
+            log.err('Failed to do identity impersonation')
```

Comment:
Oh, and two more things:

- you want to pass `err` as the first argument to `log.err`. That function takes a failure or exception as its first argument, and the message as its second argument.
- You should `return err` from `_log_failed_auth`, because returning None will cause it to consider the Deferred "successful", which will cause the next callback to run when it wouldn't before this change.